ntbs-service/Pages/Notifications/Overview.cshtml.cs

```diff
@@ -48,5 +51,10 @@ public async Task<IActionResult> OnPostCreateLinkAsync()
 
             return RedirectToPage("/Notifications/Edit/PatientDetails", new { id = linkedNotification.NotificationId });
         }
+
+        public async Task GetAlertsAsync()

```

Comment:
With reference to my other comment on NotificationModelBase - this being defined here is probably supporting the move to get the collection in here rather than in the base.

Reply:
Ohhhhhhhhh, it's in the base, so the layouts can have it...

Reply:
In that case I might be inclined to define this method in the base aswell